toronto/src/pages/Post.jsx

```diff
@@ -18,16 +16,17 @@ const Post = () => {
   });
```

Comment:
`postData` state의 초기 값을 아래처럼 변경하면 어떨까요?! 그러면 `click`시 `formData`를 보내는 부분에서 단축 연산을 사용할 수 있을 것 같습니다!
```suggestion
const Post = () => {
  const [postData, setPostData] = useState({
    postTitle: '',
    postContent: '',
    agreeContent: '',
    disagreeContent: '',
    image: null,
  });
```